### PR TITLE
Collision covariance matrix: converter with fix

### PR DESCRIPTION
**Pull request by ddobrigk**
* adds collision converter task to swap XZ and YY elements of collision covariance matrix 
* more information on the problem this intends to fix is [here](https://indico.cern.ch/event/1219192/contributions/5128952/attachments/2542060/4376840/DDChinellato-MondayMeeting-7Nov2022-1.pdf) 
* goes together with https://github.com/AliceO2Group/AliceO2/pull/10424

### CODE REVIEW

**Comment by ddobrigk** [comment]
Since https://github.com/AliceO2Group/AliceO2/pull/10424 is ready, I think this can be merged now - correct? I will wait until ~end of day and then try to self-approve (via auto-merge)...

**Comment by ddobrigk** [comment]
Ah, no - we need a new O2 version to build against...

**Comment by jgrosseo** [comment]
This should have happened with the nightly automatically.
@ktf @TimoWilken was the tag not moved this morning to the nightly as usual?

File: Common/TableProducer/collisionConverter.cxx
Comment (jgrosseo) [inline Common/TableProducer/collisionConverter.cxx:22]: Probably one could subscribe to the entire table and do the loop in process. Less function calls...

**Comment by TimoWilken** [comment]
@jgrosseo it did, at 10:09; the build/O2Physics/o2 check finished at 11:03 and picked up today's nightly (nightly-20221109) according to the build log.

**Comment by jgrosseo** [comment]
Ah yes, the failure is from before and now it is green

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**
